src/kiwixapp.cpp

```diff
@@ -444,3 +445,11 @@ void KiwixApp::updateNameMapper()
 {
   m_nameMapper.update();
 }
+
+void KiwixApp::printVersions(std::ostream& out) {
+  out << version.toStdString() << std::endl;
+  out << "+ libqt " << QT_VERSION_STR << " (" << qVersion() << ")" << std::endl << std::endl;
```

Comment:
IMHO, this line will be confusing. The previous way of reporting the build-time and run-time versions was clearer.

Reply:
@veloman-yunkan Better now?